src/dbus/server/dbus_thread_object_rcp.cpp

```diff
@@ -150,7 +152,8 @@ otbrError DBusThreadObjectRcp::Init(void)
                    std::bind(&DBusThreadObjectRcp::LeaveNetworkHandler, this, _1));
     RegisterMethod(OTBR_DBUS_THREAD_INTERFACE, OTBR_DBUS_SET_NAT64_ENABLED_METHOD,
                    std::bind(&DBusThreadObjectRcp::SetNat64Enabled, this, _1));
-
+    RegisterMethod(OTBR_DBUS_THREAD_INTERFACE, OTBR_DBUS_SET_ADMIN_PASSCODE_ENABLED_METHOD,
```

Comment:
I think this looks like a `PropertySet` method. So you may consider registering it as a `SetPropertyHandler`. For example, https://github.com/openthread/ot-br-posix/blob/main/src/dbus/server/dbus_thread_object_rcp.cpp#L157

And the implementation of `SetPropertyHandler` will be even simpler. Current approach is fine. It's up to you if to update it.

Reply:
Modify according to the suggestion, thanks!